test/cmd/builds.sh

```diff
@@ -25,8 +25,6 @@ os::test::junit::declare_suite_start "cmd/builds"
 
 os::cmd::expect_success 'oc new-build centos/ruby-22-centos7 https://github.com/openshift/ruby-hello-world.git'
 os::cmd::expect_success 'oc get bc/ruby-hello-world'
-os::cmd::expect_success 'cat "${OS_ROOT}/Dockerfile" | oc new-build -D - --name=test'
```

Comment:
the test is useful, we should keep it ... just point it to a different Dockerfile, thx for catching that